File: Ejercicio 1/MaximoArray.java
// MaximoArray.java
// Este programa calcula el máximo de un array de números en Java

public class MaximoArray {
    // Método que recibe un array de enteros y retorna el máximo
    public static int maximoArray(int[] arr) {
        int max = arr[0];
        for (int i = 1; i < arr.length; i++) {
            if (arr[i] > max) {
                max = arr[i];
            }
        }
        return max;
    }

    public static void main(String[] args) {
        int[] numeros = {3, 7, 2, 9, 4};
        System.out.println("El máximo es: " + maximoArray(numeros));
    }
} 

File: Ejercicio 1/maximoArray.js
// maximoArray.js
// Este script calcula el máximo de un array de números en JavaScript

// Definimos una función que recibe un array y retorna el máximo
function maximoArray(arr) {
    // Usamos el método Math.max junto con el spread operator
    return Math.max(...arr);
}

// Ejemplo de uso
const numeros = [3, 7, 2, 9, 4];
console.log("El máximo es:", maximoArray(numeros)); 